i3lock.c

```diff
@@ -1446,31 +1635,41 @@ static void raise_loop(xcb_window_t window) {
 /*
  * Loads an image from the given path. Handles JPEG and PNG. Returns NULL in case of error.
  */
-cairo_surface_t* load_image(char* image_path) {
+cairo_surface_t *load_image(enum IMAGE_FORMAT format) {
     cairo_surface_t *img = NULL;
     JPEG_INFO jpg_info;
-
-    if (image_raw_format != NULL && image_path != NULL) {
-        /* Read image. 'read_raw_image' returns NULL on error,
-         * so we don't have to handle errors here. */
-        img = read_raw_image(image_path, image_raw_format);
-    } else if (verify_png_image(image_path)) {
-        /* Create a pixmap to render on, fill it with the background color */
-        img = cairo_image_surface_create_from_png(image_path);
-    } else if (file_is_jpg(image_path)) {
-        DEBUG("Image looks like a jpeg, decoding\n");
-        unsigned char* jpg_data = read_JPEG_file(image_path, &jpg_info);
+    unsigned char *jpg_data;
+
+    /* Create a pixmap to render on, fill it with the background color */
```

Comment:
This comment isn't supposed to be here, is it?

Reply:
Yeah.. it was an accident :)